src/scaffoldmaker/annotation/heart_terms.py

```diff
@@ -0,0 +1,65 @@
+"""
+Common resource for heart annotation terms.
+"""
+
+# convention: preferred name, preferred id, followed by any other ids and alternative names
+heart_terms = [
```

Comment:
Fischer rat heart data also includes epicardium of left and right auricle. Not sure if we need to add these to the scaffold too?

Reply:
They aren't in interlex, but are in FMA. I'll add them as they'll help.

Reply:
Added.